configure

```diff
@@ -10553,13 +10553,17 @@ if test x"${build}" = x"${host}" ; then
   LDFLAGS_FOR_BUILD=${LDFLAGS_FOR_BUILD-${LDFLAGS}}
 fi
 
-# On Canadian crosses, we'll be searching the right directories for
-# the previously-installed cross compiler, so don't bother to add
-# flags for directories within the install tree of the compiler
-# being built; programs in there won't even run.
-if test "${build}" = "${host}" && test -d ${srcdir}/gcc; then
+if test -d ${srcdir}/gcc; then
```

Comment:
Is this change something that should be sent upstream?

Reply:
It was already sent upstream and merged (will be part of GCC 15).

https://github.com/gcc-mirror/gcc/commit/dff727b2c28c52e90e0bd61957d15f907494b245